cibuild.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+echo Changing mono snapshot
+. mono-snapshot mono/20150316155603
+if [ $? -ne 0 ]; then
```

Comment:
Congrats for being the first bash script in history to properly think about error handling.